src/plots/gl3d/layout/axis_attributes.js

```diff
@@ -14,6 +14,7 @@ var extendFlat = require('../../../lib/extend').extendFlat;
 
 
 module.exports = {
+    visible: axesAttrs.visible,
```

Comment:
`scene.?axis.visible: false` would be a nice option too. So I don't think it's worth changing the logic. Opening a new issue about implementing  `?axis.visible` in 3D will suffice.